packages/stitching-directives/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@graphql-tools/stitching-directives",
-  "version": "2.3.30",
+  "version": "2.3.31",
```

Comment:
Could you fix these unrelated diffs?

Reply:
I am sorry I am not sure what I should do, I never did this before.

Reply:
sorry about that, had a collegue help out, it should be ok now, let me know